src/main/java/de/zalando/aruha/nakadi/controller/EventTypeController.java

```diff
@@ -67,12 +73,13 @@ public EventTypeController(final EventTypeRepository eventTypeRepository, final
 
         try {
             validateSchema(eventType);
+            validatePartitioningStrategy(eventType);
             eventTypeRepository.saveEventType(eventType);
             topicRepository.createTopic(eventType.getName());
             return status(HttpStatus.CREATED).build();
-        } catch (final InvalidEventTypeException e) {
-            return create(e.asProblem(), nativeWebRequest);
-        } catch (final DuplicatedEventTypeNameException e) {
+        } catch (final InvalidEventTypeException | NoSuchPartitioningStrategyException |
+                DuplicatedEventTypeNameException e) {
+            LOG.warn(e.getMessage(), e);
```

Comment:
I'm confused about when to use warning and when to use debug, for example. Because this seems to something for debug as I understand it. So I would like to know your opinion.